### DISCUSSION TITLE
Color of the program window

### DISCUSSION THREAD

**Discussion by Eave64** [Monitor221hz/Pandora-Behaviour-Engine-Plus#305] [Q&A] [answered]
Is there a way to change the color of the display window? The latest version, at least for me, has black lettering on dark purple background. Any help would be appreciated
![Pandora_Behaviour_Engine_v2 3 0-beta_color](https://github.com/user-attachments/assets/917f3e89-3f14-4ede-b000-74b00f411c3d)

**Comment by Inzania2016** [comment]
I saw a branch with an update to that but yeah it's hard to see.

**Comment by aarchiee1965** [comment]
Glad its not just me. I thought id screwed up the install. just a font change to white will do the trick.

**Comment by aarchiee1965** [comment]
> Is there a way to change the color of the display window? The latest version, at least for me, has black lettering on dark purple background. Any help would be appreciated ![Pandora_Behaviour_Engine_v2 3 0-beta_color](https://private-user-images.githubusercontent.com/54476091/363761801-917f3e89-3f14-4ede-b000-74b00f411c3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU5OTA1MzUsIm5iZiI6MTcyNTk5MDIzNSwicGF0aCI6Ii81NDQ3NjA5MS8zNjM3NjE4MDEtOTE3ZjNlODktM2YxNC00ZWRlLWIwMDAtNzRiMDBmNDExYzNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEwVDE3NDM1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzMjNkZGIwZjY3MWRkMzQzNzM3OGFhNDRiODVmOTA2ZGM4MTJjZTM2YjE5NGQ1MTQ0ZGQwNTg3YmRjY2E5OWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eVG6VKnU9FOS03uBOgAUEf4FSG_Q1mfvmEkhHNZJ82E)

Found a fix. You need to run windows mode and default app mode in dark. On you pc under settings, personalisation, colours. Guessing the mod author runs his pc like that so it was built visually like that.

**Reply by aarchiee1965** [reply]
> > Is there a way to change the color of the display window? The latest version, at least for me, has black lettering on dark purple background. Any help would be appreciated ![Pandora_Behaviour_Engine_v2 3 0-beta_color](https://private-user-images.githubusercontent.com/54476091/363761801-917f3e89-3f14-4ede-b000-74b00f411c3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU5OTA1MzUsIm5iZiI6MTcyNTk5MDIzNSwicGF0aCI6Ii81NDQ3NjA5MS8zNjM3NjE4MDEtOTE3ZjNlODktM2YxNC00ZWRlLWIwMDAtNzRiMDBmNDExYzNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEwVDE3NDM1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzMjNkZGIwZjY3MWRkMzQzNzM3OGFhNDRiODVmOTA2ZGM4MTJjZTM2YjE5NGQ1MTQ0ZGQwNTg3YmRjY2E5OWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eVG6VKnU9FOS03uBOgAUEf4FSG_Q1mfvmEkhHNZJ82E)
> 
> Found a fix. You need to run windows mode and default app mode in dark. On you pc under settings, personalisation, colours. Guessing the mod author runs his pc like that so it was built visually like that.

Edit. Just needs default app mode to be dark.

**Reply by Sumerianhaze** [reply]
Default app mode? do you mean in the windows list of default apps? Using MO2 Pandora is installed as a mod not an app.

Switching windows to dark mode each time a Pandora run is required is a temporary work around and at least makes the UI usable.

As I would guess over 90% of users have windows in Light mode this really could do with being addressed.

**Reply by Monitor221hz** [reply, marked as answer]
> As I would guess over 90% of users have windows in Light mode this really could do with being addressed.

The release notes of 2.3.5-beta state clearly that this problem has been fixed. There's also a merged pull request that indicates so #300.

**Comment by Eave64** [comment]
Not an option for me, not running windows in dark mode for just one program.
